File: src/main/java/cn/spark/study/SparkSQL.java
package cn.spark.study;

import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaSparkContext;

/**
 * @Auther: huzhibin_zz
 * @Date: 2022/8/30
 * @Description :
 */
public class SparkSQL {
    static String DATAPATH="E:\\code\\IdeaProject\\studySpark\\src\\main\\resources\\DataFile\\";
    static String OUTPATH="E:\\code\\IdeaProject\\studySpark\\src\\main\\resources\\OUT\\";
    static SparkConf conf = new SparkConf().setAppName("studyRdd").setMaster("local");
    //必须使用JavaSparkContext
    static JavaSparkContext sc=new JavaSparkContext(conf);

    public static void main(String[] args) {

    }
}

File: src/main/scala/cn/spark/study01/SparkSQL.scala
package cn.spark.study01

import org.apache.spark.{SparkConf, SparkContext}

/**
 * @Auther: huzhibin_zz
 * @Date: 2022/8/30
 * @Description :
 *
 */
object SparkSQL {
  def main(args: Array[String]): Unit = {
    val cof = new SparkConf().setMaster("local[3]").setAppName("RddStudy01")
    val sc = new SparkContext(cof)
  }

}
